components/display/include/display.hpp

```diff
@@ -54,7 +48,8 @@ class Display : public BaseComponent {
     size_t width;             /**< Width of th display, in pixels. */
     size_t height;            /**< Height of the display, in pixels. */
     size_t pixel_buffer_size; /**< Size of the display buffer in pixels. */
-    flush_fn flush_callback;  /**< Function provided to LVGL for it to flush data to the display. */
+    lv_display_flush_cb_t
```

Comment:
Can we do a `using flush_cb_t = lv_display_flush_cb_t` in this class so that if this changes in the future, there are fewer changes we need to make :)